src/tailwind/tokens-output/dark.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
So I think instead of duplicating what we need to do is:

- Add git ignore for tokens-output under tailwind
- On package json, add pre-publish script to copy tokens-output from theming folder into tailwind folder

That way we aren't duplicating the token output

Reply:
made changes